loky/process_executor.py

```diff
@@ -853,7 +853,7 @@ class ProcessPoolExecutor(_base.Executor):
 
     def __init__(self, max_workers=None, job_reducers=None,
                  result_reducers=None, timeout=None, context=None,
-                 initializer=None, initargs=()):
+                 initializer=None, initargs=(), env={}):
```

Comment:
Please do not use mutable datastructures as the default value for kwargs. This can lead to very subtle bugs.

Instead pass `env=None` and use a construct such as:

```python
env = {} if env is None else env
```

in the body of the method/function.